src/main/java/core/basesyntax/service/impl/ReadFromFileServiceImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReadFromFileService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReadFromFileServiceImpl implements ReadFromFileService {
+    @Override
+    public List<String> redData(String fromFileName) {
+        List<String> dataFromFileList = new ArrayList<>();
+        File file = new File(fromFileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(file))) {
```

Comment:
```suggestion
        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(filePath))) {
```

Reply:
I use referense "file" in exception message. So i need it. To get full information about the problem.